raiden-dapp/src/components/navigation/Transfer.vue

```diff
@@ -296,9 +296,10 @@ export default class Transfer extends Mixins(BlockieMixin, NavigationMixin) {
     &__form {
       background-color: $transfer-form-color;
       border-radius: 15px;
-      height: 175px;
```

Comment:
Would it be easy to increase the fixed size to cover two lines of text instead?
The reason it was originally fixed was to avoid having the UI jump every time extra vertical space was needed.

Reply:
The error message in that case can be quite long do you think we should reserve the blank space for it or leave it with the jump?

<img width="549" alt="Screenshot 2020-05-15 at 14 52 00" src="https://user-images.githubusercontent.com/3169205/82052356-cbc5a580-96bb-11ea-9cca-53ac1a9b5e0e.png">

I feel like both is not ideal 😓

Reply:
Indeed also we should make sure that the message never goes to more than two lines. but it is hard to ensure. maybe we can keep it with jumping initially and discuss an alternative solution. @taleldayekh maybe adding an error indicator in the input like an error icon that gives you the erron on tooltip?